From 08e7244811b18b9a332f2191bff342722b6fcd01 Mon Sep 17 00:00:00 2001
From: Hanna Hawa <hannah@marvell.com>
Date: Tue, 31 May 2016 18:29:19 +0300
Subject: [PATCH 478/538] fix: dts: a70x0: update SPI partitions for
 Armada-70x0

- Change the name of U-Boot partition for boot, because a70x0
  support more than one bootloader (U-Boot, UEFI)
- Add new partition with 1M for boot usage (BR bug)

Change-Id: I970cfff1b160d1db54888347e582a6ca7407a039
Signed-off-by: Hanna Hawa <hannah@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/30233
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Haim Boot <hayim@marvell.com>
Reviewed-by: Neta Zur Hershkovits <neta@marvell.com>
---
 arch/arm64/boot/dts/marvell/armada-7040-db.dtsi | 12 +++++++-----
 1 file changed, 7 insertions(+), 5 deletions(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-7040-db.dtsi b/arch/arm64/boot/dts/marvell/armada-7040-db.dtsi
index 404c2e7..8485b62 100644
--- a/arch/arm64/boot/dts/marvell/armada-7040-db.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-7040-db.dtsi
@@ -99,15 +99,17 @@
 					compatible = "jedec,spi-nor";
 					reg = <0x0>;
 					spi-max-frequency = <20000000>;
-
 					partition@0 {
-						label = "U-Boot";
+						label = "boot";
 						reg = <0x0 0x200000>;
 					};
-
-					partition@400000 {
+					partition@200000 {
 						label = "Filesystem";
-						reg = <0x200000 0xe00000>;
+						reg = <0x200000 0xd00000>;
+					};
+					partition@d00000 {
+						label = "boot_2nd";
+						reg = <0xd00000 0x100000>;
 					};
 				};
 			};
-- 
1.9.1

